packages/dataclass-editor/src/components/dataclass/data/dataclass-field-utils.test.ts

```diff
@@ -0,0 +1,32 @@
+import { DataClassFieldModifier, type DataClassField } from './dataclass';
+import { isPersistent } from './dataclass-field-utils';
+
+let dataClassField: DataClassField;
+
+beforeEach(() => {
+  dataClassField = {
+    name: 'DataClassField',
+    type: 'String',
+    comment: 'comment',
+    modifiers: [],
+    annotations: []
+  };
+});
+
+describe('dataclass-field-utils', () => {
+  describe('isPersistent', () => {
+    test('true', () => {
+      dataClassField.modifiers = [DataClassFieldModifier.PERSISTENT];
+      expect(isPersistent(dataClassField)).toBeTruthy();
```

Comment:
And why do you need such a fully coded object if you only check the modifiers field? Otherwise we need to adjust the test object if a new attribute is get added...

In a test case it is definitely fine to trick typescript e.g. like this:
```ts
const dataClassField = { modifiers: [DataClassFieldModifier.PERSISTENT] } as DataClassField;
expect(isPersistent(dataClassField)).toBeTruthy();
```